crates/bw/src/main.rs

```diff
@@ -62,6 +68,39 @@ enum ItemCommands {
     Create {},
 }
 
+#[derive(Subcommand, Clone)]
+enum GeneratorCommands {
+    Password(PasswordGeneratorArgs),
+    Passphrase {},
+}
+
+#[derive(Args, Clone)]
+struct PasswordGeneratorArgs {
+    #[arg(short = 'l', long, action, help = "Include lowercase characters (a-z)")]
+    lowercase: bool,
+
+    #[arg(short = 'u', long, action, help = "Include uppercase characters (A-Z)")]
+    uppercase: bool,
+
+    #[arg(short = 'n', long, action, help = "Include numbers (0-9)")]
+    numbers: bool,
+
+    #[arg(
+        short = 's',
+        long,
+        action,
+        help = "Include special characters (!@#$%^&*)"
+    )]
+    special: bool,
+
+    #[arg(
+        long,
+        default_value = "16",
+        help = "Minimum length of generated password"
```

Comment:
There's no minimum and maximum, the length of the generated password is exactly what's provided, so the comment should probably be "Length of generated password" right?

Reply:
Yep, codepilot 😠